config/daemonconfig.go

```diff
@@ -61,8 +61,9 @@ type DaemonConfig struct {
 }
 
 type MirrorConfig struct {
-	Host    string            `json:"host,omitempty"`
-	Headers map[string]string `json:"headers,omitempty"`
+	Host        string            `json:"host,omitempty"`
+	Headers     map[string]string `json:"headers,omitempty"`
+	AuthThrough bool              `json:"auth_through,omitempty"`
```

Comment:
This option should also be available for `proxy` mode, can we move it to `device.backend.config.auth_through`?